### PR TITLE
closes #11 find and set up test framework

### LINKED ISSUE #11: Find and set up test framework
**Is your feature request related to a problem? Please describe.**
A way to automatically test the code is needed.

**Describe the solution you'd like**
Create an automatic pipeline in Github using a test framework. This framework also needs to be defined.

**Describe alternatives you've considered**
No real alternatives.

**Additional context**

### PR DESCRIPTION
**Pull request by rajtarn**

### CODE REVIEW

**Comment by rajtarn** [comment]
https://github.com/PUM04/pum-04/pull/28#pullrequestreview-1358009954

**Review by AntonOstman — CHANGES_REQUESTED** [review]
Does not look like this was merged with main before the PR was created. Otherwise I don't see any issues, Nice work!

**Review by AntonOstman — APPROVED** [review]
Nice!

**Review by ViktorAndersson99 — APPROVED** [review]
Nice work!

File: src/components/Example.tsx
Comment (ELEMENTLHERO) [inline src/components/Example.tsx:1]: Varför tar ni bort WASM Calculator? Är det för den är irrelevant och WASM verkar funka nu? Inget fel bara undrade

**Pull request merged**